react/src/components/organisms/AccordionWrapper/index.js

```diff
@@ -17,8 +17,8 @@ const AccordionWrapper = (props) => {
           return clone;
         }
         return(
-          /* eslint-disable no-console */
-          console.log(`Warning! You cannot pass a ${child.type.name} child to AccordionWrapper`)
+          /* eslint-disable-next-line no-console */
+          console.warn('Warning! You can only pass an AccordionItem child to AccordionWrapper.')
```

Comment:
this is only checking if there's a child passed and but not actually checking if it's an AccordionItem, is that the intention?

Reply:
@petermallett why do we need to pass non accordionItem into accordionWrapper in EEC?